apps/explorer/src/pages/transaction-result/TxResultHeader.tsx

```diff
@@ -32,6 +32,7 @@ function TxAddressHeader({ data }: { data: TxResultState }) {
         // TODO: use a different icon
         ChangeEpoch: CallTypeIcon,
         TransferSui: TransferObjectTypeIcon,
+        Pay: TransferObjectTypeIcon,
```

Comment:
here each time we introduce a new tx type, this will break unless we also change here, shall we have some default value here?